### DISCUSSION TITLE
no configuration file provided: not found

### DISCUSSION THREAD

**Discussion by RealMrCactus** [postalserver/postal#2955] [Help with using Postal]
no configuration file provided: not found How do i fix this it tried sudo didnt work

**Comment by willpower232** [comment]
I'm not trying to be sarcastic but did you set up a configuration file?

**Reply by RealMrCactus** [reply]
it never said to do that in the installation guide as far as im aware

**Reply by willpower232** [reply]
![image](https://github.com/postalserver/postal/assets/1619102/b89d8271-ddad-401c-bf07-9725c4140d16)

**Reply by vongohren** [reply]
Im acctually getting this now, and I did run boostrap and verify the changes. So not sure why its failing with this error

**Comment by RealMrCactus** [comment]
> ![image](https://private-user-images.githubusercontent.com/1619102/328459198-b89d8271-ddad-401c-bf07-9725c4140d16.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUzNDQ3NzcsIm5iZiI6MTcxNTM0NDQ3NywicGF0aCI6Ii8xNjE5MTAyLzMyODQ1OTE5OC1iODlkODI3MS1kZGFkLTQwMWMtYmYwNy05NzI1YzQxNDBkMTYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUxMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MTBUMTIzNDM3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzA5ZjNmMzUwNDdiMjBkMDcwZWJmY2VmZDg4OWRlZTlhYTliY2YxYjkyZTMyZWY3ZjkwZDNhOTk5MzE1NGViZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.EPIC8R6aFaU31rwebPAOBwtAG9DVWZnFIUmXh2XEgsw)

I had
